src/main/resources/javascript/locales/en.json

```diff
@@ -190,7 +190,7 @@
         "createNewContent": {
           "menu": "New content",
           "contentOfType": "New {{typeName}}",
-          "labelModal": "SELECT CONTENT TYPE",
+          "labelModal": "Select Content Type",
```

Comment:
We need to change the labels in other languages as well

Reply:
Good point! I've now updated French and German as well.